### PR TITLE
Practices: Adds Research Workshop

### PR DESCRIPTION
**Pull request by joemoore**
As requested by @fanibahar

### CODE REVIEW

File: content/practices/research-workshop/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Research Workshop"
```
Comment (fanibahar) [inline content/practices/research-workshop/index.md:2]: Might worth to change it to "User Research Introduction" or "Research Intro & De-risking Assumptions: Exploratory Research"
As the content describe the introduction to user research

File: content/practices/research-workshop/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+linkTitle: "Research Workshop"
+description: "Level-set on user research best practices"
+tags: ["Discovery"]
+length: "45 mins - 1 hour"
+participants: "Core team"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-05-14"
+why: 
+- Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This workshop offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.
```
Comment (fanibahar) [inline content/practices/research-workshop/index.md:12]: Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This session offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.

File: content/practices/research-workshop/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+None at the moment. 
+
+## Real World Examples
+
+![Research Workshop Example 1](/images/practices/research-workshop/example-1.jpg)
+
+![Research Workshop Example 2](/images/practices/research-workshop/example-2.jpg)
+
+## Recommended Reading
+None at the moment
```
Comment (fanibahar) [inline content/practices/research-workshop/index.md:128]: Recommended reading:
Lean UX (https://www.oreilly.com/library/view/lean-ux/9781449366834/) by Jeff Gothelf and Josh Seiden
UX For Lean Startup (https://www.amazon.com/UX-Lean-Startups-Experience-Research/dp/1449334911) by Eric Ries

**Review by fanibahar — COMMENTED** [review]
WIP: Added couple of comment in this workshop

File: content/practices/research-workshop/index.md (hunk 2 above)
Comment (joemoore) [inline content/practices/research-workshop/index.md:12]:
```suggestion
- Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This session offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.
```

File: content/practices/research-workshop/index.md (hunk 1 above)
Comment (joemoore) [inline content/practices/research-workshop/index.md:2]:
```suggestion
title: "User Research Introduction Session"
```

File: content/practices/research-workshop/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+tags: ["Discovery"]
+length: "45 mins - 1 hour"
+participants: "Core team"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-05-14"
+why: 
+- Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This workshop offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.
+when:
+- Given the importance of research early in the project, this workshop is typically done as soon as possible after Kickoff in advance of any stakeholder interviews or exploratory user research.
```
Comment (joemoore) [inline content/practices/research-workshop/index.md:14]:
```suggestion
- Given the importance of research early in the project, this session is typically done as soon as possible after Kickoff in advance of any stakeholder interviews or exploratory user research.
```

File: content/practices/research-workshop/index.md (hunk 3 above)
Comment (joemoore) [inline content/practices/research-workshop/index.md:128]:
```suggestion
[Lean UX](https://www.oreilly.com/library/view/lean-ux/9781449366834/) by Jeff Gothelf and Josh Seiden

[UX For Lean Startup](https://www.amazon.com/UX-Lean-Startups-Experience-Research/dp/1449334911) by Eric Ries
```

File: content/practices/research-workshop/index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Research Workshop"
+linkTitle: "Research Workshop"
```
Comment (joemoore) [inline content/practices/research-workshop/index.md:3]:
```suggestion
linkTitle: "User Research Introduction Session"
```

**Review by joemoore — COMMENTED** [review]
Incorporating feedback

File: content/practices/research-workshop/index.md (hunk 3 above)
Comment (joemoore) [inline content/practices/research-workshop/index.md]:
```suggestion
[Lean UX](https://www.oreilly.com/library/view/lean-ux/9781449366834/) by Jeff Gothelf and 
Josh Seiden

[UX For Lean Startup](https://www.amazon.com/UX-Lean-Startups-Experience-Research/dp/1449334911) by Eric Ries
```

File: content/practices/research-workshop/index.md (hunk 1 above)
Comment (joemoore) [reply content/practices/research-workshop/index.md]: Updated.

File: content/practices/research-workshop/index.md (hunk 2 above)
Comment (joemoore) [reply content/practices/research-workshop/index.md]: Updated.

File: content/practices/research-workshop/index.md (hunk 3 above)
Comment (joemoore) [reply content/practices/research-workshop/index.md]: Added

Comment (joemoore) [reply content/practices/research-workshop/index.md]: Added.

Comment (joemoore) [reply content/practices/research-workshop/index.md]: Added.

File: content/practices/research-workshop/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (joemoore) [reply content/practices/research-workshop/index.md]: Updated.

File: content/practices/user-research-introduction-session/index.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+   - Usability only: create pass/fail table to quickly see which aspects of the design worked and which did not
+   - See Research Synthesis for more information
+   
+   {{< callout >}}
+   Tip: Drawing these four steps on the board showing a progression can be really helpful to visualize how they flow together
+   {{< /callout >}}
+   
+1. Share **biases** to be aware of and key **“dos”** and **“don’ts”** when conducting research by printing out or projecting this **Research Dos & Don'ts Guide** 
+
+   ***EDITOR NOTE: The above Research Dos & Don'ts Guide is an internal google doc. Please change, publish the doc, or find an alternative***
```
Comment (joemoore) [inline content/practices/user-research-introduction-session/index.md:83]: @fanibahar Is there a different resource we can link to here, or can we convert the internal doc to a companion blog post, PDF, etc.?

Comment (joemoore) [reply content/practices/user-research-introduction-session/index.md]: I added it as a page. See update.

**Comment by joemoore** [comment]
Ready for final review and merge.

**Pull request merged**